.babelrc

```diff
@@ -1,9 +1,5 @@
 {
-  "plugins": [
-    "syntax-export-extensions",
-    "transform-export-extensions",
-    "glamorous-displayname"
-  ],
+  "plugins": ["syntax-export-extensions", "transform-export-extensions"],
```

Comment:
This is the babel plugin that supports `export from` syntax. (from earlier convo) I would love to remove it.  The more I babel, the less I want to.